istio/istio/base_v3/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base/kf-istio-resources.yaml
```

Comment:
This is same as istio/istio/base/kustomize?

Reply:
Right, they are the same.